docs/pages/access-controls/login-rules/kubernetes.mdx

```diff
@@ -0,0 +1,248 @@
+---
+title: Deploy Login Rules using Kubernetes Operator (Preview)
+description: Use Teleport's Kubernetes Operator to deploy Login Rules to your cluster
+---
+
+<Admonition type="warning" title="Preview">
+  Login Rules and the Teleport Kubernetes Operator are currently in Preview mode.
+</Admonition>
+
+This guide will explain how to:
+
+- Use Teleport's Kubernetes Operator to deploy Login Rules to your Teleport cluster
+- Edit deployed Login Rules with `kubectl`
+
+This guide is applicable if you self-host Teleport in Kubernetes using the
+`teleport-cluster` Helm chart.
+
+## Prerequisites
+
+- A Kubernetes cluster (with or without `teleport-cluster` Helm chart already deployed)
+- [Helm](https://helm.sh/docs/intro/quickstart/)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/)
+
+Validate Kubernetes connectivity by running the following command:
+
+```code
+$ kubectl cluster-info
+# Kubernetes control plane is running at https://127.0.0.1:6443
+# CoreDNS is running at https://127.0.0.1:6443/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
+# Metrics-server is running at https://127.0.0.1:6443/api/v1/namespaces/kube-system/services/https:metrics-server:https/proxy
+```
+
+<Admonition type="tip">
+  Users wanting to experiment locally with the Operator can use [minikube](https://minikube.sigs.k8s.io/docs/start/)
+  to start a local Kubernetes cluster:
+  ```code
+  $ minikube start
+  ```
+</Admonition>
+
+## Step 1/3. Install the Teleport Operator
+
+Follow Step 1 of the
+[Teleport operator guide](../../management/guides/teleport-operator.mdx#step-13-install-teleport-cluster-helm-chart-with-the-operator)
+to install the Teleport Operator in your Kubernetes cluster.
```

Comment:
The need to follow instructions on a separate page should probably be listed under Prerequisites.